From b8d3c7bcb147a42010173bdffabbbfee6ead8824 Mon Sep 17 00:00:00 2001
From: KhasMek <Boushh@gmail.com>
Date: Tue, 4 Feb 2014 22:59:52 -0700
Subject: [PATCH 15/28] Rearrange morekeys

  Some of the symbols such as *,!,-,+ were moved in kitkat,
  this moves the moreKeys to be inline with the moves.

Change-Id: I5aff699b268dd32a5b1e4539163e6ec153021f0a
---
 java/res/xml/rowkeys_qwerty2_right4.xml | 16 ++++++++--------
 java/res/xml/rowkeys_qwerty3_left4.xml  |  4 ++--
 java/res/xml/rowkeys_qwerty3_right3.xml |  4 ++--
 3 files changed, 12 insertions(+), 12 deletions(-)

diff --git a/java/res/xml/rowkeys_qwerty2_right4.xml b/java/res/xml/rowkeys_qwerty2_right4.xml
index 1ee8fd570..c936280d7 100644
--- a/java/res/xml/rowkeys_qwerty2_right4.xml
+++ b/java/res/xml/rowkeys_qwerty2_right4.xml
@@ -23,22 +23,22 @@
 >
     <Key
         latin:keySpec="h"
-        latin:keyHintLabel="*"
-        latin:additionalMoreKeys="*"
+        latin:keyHintLabel="-"
+        latin:additionalMoreKeys="-"
         latin:moreKeys="!text/morekeys_h" />
     <Key
         latin:keySpec="j"
-        latin:keyHintLabel="-"
-        latin:additionalMoreKeys="-"
+        latin:keyHintLabel="+"
+        latin:additionalMoreKeys="+"
         latin:moreKeys="!text/morekeys_j" />
     <Key
         latin:keySpec="k"
-        latin:keyHintLabel="+"
-        latin:additionalMoreKeys="+"
+        latin:keyHintLabel="("
+        latin:additionalMoreKeys="("
         latin:moreKeys="!text/morekeys_k" />
     <Key
         latin:keySpec="l"
-        latin:keyHintLabel="_"
-        latin:additionalMoreKeys="_"
+        latin:keyHintLabel=")"
+        latin:additionalMoreKeys=")"
         latin:moreKeys="!text/morekeys_l" />
 </merge>
diff --git a/java/res/xml/rowkeys_qwerty3_left4.xml b/java/res/xml/rowkeys_qwerty3_left4.xml
index 67b0e2743..8dc9d41c5 100644
--- a/java/res/xml/rowkeys_qwerty3_left4.xml
+++ b/java/res/xml/rowkeys_qwerty3_left4.xml
@@ -23,8 +23,8 @@
 >
     <Key
         latin:keySpec="z"
-        latin:keyHintLabel="!"
-        latin:additionalMoreKeys="!"
+        latin:keyHintLabel="*"
+        latin:additionalMoreKeys="*"
         latin:moreKeys="!text/morekeys_z" />
     <Key
         latin:keySpec="!text/keyspec_x"
diff --git a/java/res/xml/rowkeys_qwerty3_right3.xml b/java/res/xml/rowkeys_qwerty3_right3.xml
index fbcf62622..464602907 100644
--- a/java/res/xml/rowkeys_qwerty3_right3.xml
+++ b/java/res/xml/rowkeys_qwerty3_right3.xml
@@ -27,8 +27,8 @@
         latin:additionalMoreKeys=";" />
     <Key
         latin:keySpec="n"
-        latin:keyHintLabel="/"
-        latin:additionalMoreKeys="/"
+        latin:keyHintLabel="!"
+        latin:additionalMoreKeys="!"
         latin:moreKeys="!text/morekeys_n" />
     <Key
         latin:keySpec="m"
-- 
2.17.1

